src/packages/admin-ui-components/src/detail-panel/component.tsx

```diff
@@ -71,9 +71,9 @@ const getField = ({ field }: { field: EntityField }) => {
 	}
 
 	const { enumByName } = useSchema();
-	const enumField = enumByName(field.type);
+	const enumField = enumByName(field.type.replace('[]', ''));
```

Comment:
is this replace still needed?

Reply:
Thanks for pointing it out. I have cleanup the code.